File: tablesoft-java/TableSoftModel/src/pe/edu/pucp/tablesoft/model/Urgencia.java
package pe.edu.pucp.tablesoft.model;


public class Urgencia {
    private int urgenciaId;
    private String nombre;
    private int plazoMaximo;
    private Boolean activo;

    public Urgencia() {
        
    }

    public Urgencia(String nombre, int plazoMaximo) {
        this.nombre = nombre;
        this.plazoMaximo = plazoMaximo;
    }

    public int getUrgenciaId() {
        return this.urgenciaId;
    }

    public void setUrgenciaId(int id_urgencia) {
        this.urgenciaId = id_urgencia;
    }

    public String getNombre() {
        return this.nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getPlazoMaximo() {
        return this.plazoMaximo;
    }

    public void setPlazoMaximo(int plazoMaximo) {
        this.plazoMaximo = plazoMaximo;
    }

    public Boolean getActivo() {
        return activo;
    }

    public void setActivo(Boolean activo) {
        this.activo = activo;
    }
    
    public String mostrarDatos() {
        return urgenciaId + " - " + nombre + " - " + plazoMaximo;
    }
}

File: tablesoft-net/TableSoft/TableSoft/temp/Urgencia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TableSoft.temp
{
    public class Urgencia
    {
        private int _urgenciaId;
        private String _nombre;
        private int _plazoMaximo;

        public Urgencia()
        {
        }

        public Urgencia(int id, string nombre, int plazoMaximo)
        {
            _urgenciaId = id;
            _nombre = nombre;
            _plazoMaximo = plazoMaximo;
        }

        public Urgencia(string nombre, int plazoMaximo)
        {
            _nombre = nombre;
            _plazoMaximo = plazoMaximo;
        }

        public int UrgenciaId { get => _urgenciaId; set => _urgenciaId = value; }
        public string Nombre { get => _nombre; set => _nombre = value; }
        public int PlazoMaximo { get => _plazoMaximo; set => _plazoMaximo = value; }
    }
}
